Topic1

franrai91 opened the pull request:
Terminados todos los ejercicios, con un commit por cada  uno

franrai91 committed:
Ejercicio 1

Los 4 puntos del ejercicio 1

franrai91 committed:
Ejercicio 2

franrai91 committed:
Ejercicio 3

franrai91 committed:
Ejercicio 4

franrai91 committed:
Ejercicio 6

intii commented on 01-JS-DOM-APIs/js/main.js:
```diff
@@ -5,4 +5,42 @@ function hacerFadeIn(){
+        }
+    })
+    .catch(error => {
+        document.getElementById("contenidoJson").style.background = "red";
```
Estas referencias a estilos intentá que estén siempre en los archivos de CSS

intii commented on 01-JS-DOM-APIs/js/main.js:
```diff
@@ -5,4 +5,42 @@ function hacerFadeIn(){
+            reject(Error("Hubo un error"));
+        }
+        
+        httpRqs.send();
```
En esta función la indentación te quedó toda desarreglada, fijate si lo podés arreglar

intii commented on 01-JS-DOM-APIs/js/main.js:
```diff
@@ -5,4 +5,42 @@ function hacerFadeIn(){
+    configurar.url = 'http://api.icndb.com/jokes/random';
+
+    reusable(configurar).then(response =>{
+        var objeto = JSON.parse(response);
```
Tené más cuidado con los nombres que estás usando para las cosas. `configurar` es un verbo, da la idea de que es una función. `objeto` es demasiado general. `mensajeAlerta` parece más el nombre de una variable que de una función, las funciones *hacen* algo

intii commented on 01-JS-DOM-APIs/js/main.js:
```diff
@@ -43,4 +43,20 @@ function reusable(configuracion) {
     })  
 };
+function traerConParametros(value){
+    value = document.getElementById("valorUsuario").value;
```
Por qué recibís un parámetro `value` si después lo pisás?

intii commented on 01-JS-DOM-APIs/js/main.js:
```diff
@@ -60,3 +60,26 @@ function traerConParametros(value){
+        var fila = document.createElement('tr');
+        var elemento = matriz[i].length;
+
+        for(n=0;n<elemento;n++){
```
Ojo con las variables `i` y `n`. Nunca las declaraste por lo que son globales.

intii reviewed (approved):
En líneas generales está aprobado. Te dejé un par de comentarios que me gustaría que al menos revisaras, si querés los corregís. 
Y un comentario, si podés empezar a programar en inglés, mejor. En este momento a veces tenés variables en inglés, a veces en español, y en general siempre vas a tener que programar en inglés.
También tenemos que poner foco en los nombres de las funciones y las variables

franrai91 committed:
Correcciones

-Cambie la refencia de estilo y lo pase a css
-Arregle la indentacion
-Cambie el nombre de algunas variables y funciones(para intentar hacerlas mas claras)
-Arregle la pisada de parametro value
-Solucione la declaracion de i y n en la funcion
